tests/app/test_app.py

```diff
@@ -479,10 +480,40 @@ def test_if_user_has_no_first_name_or_last_name_then_unknown_user_passed_to_case
         url = "http://localhost:5050/message/send"
         self.client.post(url, data=json.dumps(self.test_message), headers=self.headers)
         mock_case.assert_called_with('ACollectionCase', 'Unknown user')
-        mock_logger.assert_called_with('no user names in party data for id  Unknown user used in case ',
-                                       party_id='f62dfda8-73b0-4e0e-97cf-1b06327a6712')
 
+    @patch.object(InternalUserServiceMock, 'get_user_details', return_value=({"id": "f62dfda8-73b0-4e0e-97cf-1b06327a6712",
+                                                                              "emailAddress": "   ",
+                                                                              "lastName": "",
+                                                                              "telephone": "+443069990888"}))
+    @patch.object(CaseServiceMock, 'store_case_event')
+    @patch.object(message_logger, 'warning')
```

Comment:
Is this patch even needed?

Reply:
nope , not needed , removed . I cant even offer an explanation as to why on earth that was there